### DISCUSSION TITLE
TBSelenium and MobProxy or Selenium-wire?

### DISCUSSION THREAD

**Discussion by Hackswell** [webfp/tor-browser-selenium#175] [Q&A]
Does anyone know of a way to get BrowserMobProxy and/or Selenium-Wire working with TBSelenium?  I need the equivalent of HAR files generated during the sessions, just like if I were to capture in the F12 developer tools window.  Thank you!

**Comment by gunesacar** [comment]
I'd expect setting a proxy or using selenium-wire to be impossible, because Tor Browser is configured to use `tor` as a SOCKS proxy. 

Perhaps exporting HAR via an addon could work: https://github.com/firefox-devtools/har-export-trigger

Reading the discussions in the following Firefox bug report, exporting HAR via the new BiDi protocol may be possible in Firefox versions > 110, which it's [unclear whether ESRs are covered](https://bugzilla.mozilla.org/show_bug.cgi?id=1744483#c23):
- https://bugzilla.mozilla.org/show_bug.cgi?id=1744483
- https://github.com/firefox-devtools/bidi-har-export
